buildenv/jenkins/openjdk_tests

```diff
@@ -275,21 +275,39 @@ def runTest() {
             def SCM_GIT_REPO_VAL = sh(script: "echo ${SCM_GIT_REPO}", returnStdout: true).trim()
             SCM_GIT_REPO_VAL = SCM_GIT_REPO_VAL.replace("https://github.com/","git@github.com:")
 
-            sh "git clone -b ${SCM_GIT_BRANCH} ${SCM_GIT_REPO_VAL} aqa-tests"
+            def count = 0
+            retry(5) {
+                if (count > 0) {
+                    def sleep_time = 300
+                    println("Sleep ${sleep_time} secs, then retry ${count}...")
+                    sh 'sleep 300'
```

Comment:
We can directly `sleep(<SECONDS>)` https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#sleep-sleep

Reply:
Updated